LPD-25688

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

ctang3 committed:
LPD-25688 Fix Publications Timeline dropdown menu width to match Figma

ctang3 committed:
LPD-25688 Add a button to Publication Timeline dropdown menu that opens a modal showing the entire history of an entity

ctang3 committed:
LPD-25688 Implement entity history modal using FrontendDataSet

ctang3 committed:
LPD-25688 buildLang

ctang3 committed:
LPD-25688 Add feature flag

ctang3 committed:
LPD-25688 Add Playwright test

ctang3 committed:
LPD-25688 Regen tsconfig.json

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
ci:test:change-tracking-web-playwright

liferay-continuous-integration commented:
<h1>Unable to generate test report.</h1><p>Jenkins encountered the following error while generating test report:</p><pre><code>Sourced file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : TargetError : at Line: 82 : in file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : throw e ; 

Target exception: java.lang.NullPointerException
</code></pre><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/839e11f2525c8067989ea15f099940d118ea2d2d">839e11f2525c8067989ea15f099940d118ea2d2d</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/5970/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5970/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/5970/">test-portal-acceptance-pullrequest(master)#5970</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5970/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>change-tracking-web-playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/sandrodw3/liferay-portal/pull/117">sandrodw3#117</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/6389465">[master] ci:test:change-tracking-web-playwright - sandrodw3 &gt; sandrodw3 - PR#117 - 2024-09-10[01:37:27]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>6389465<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-25.liferay.com/job/publish-testray-report/12291/">publish-testray-report#12291</a><br/></div></div>

sandrodw3 commented:
ci:test:change-tracking-web-playwright

liferay-continuous-integration commented:
<h1>Unable to generate test report.</h1><p>Jenkins encountered the following error while generating test report:</p><pre><code>Sourced file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : TargetError : at Line: 82 : in file: inline evaluation of: `` 						 							import com.liferay.jenkins.results.parser.Build; 							import co . . . '' : throw e ; 

Target exception: java.lang.NullPointerException
</code></pre><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/839e11f2525c8067989ea15f099940d118ea2d2d">839e11f2525c8067989ea15f099940d118ea2d2d</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/13873/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13873/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/13873/">test-portal-acceptance-pullrequest(master)#13873</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13873/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>change-tracking-web-playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/sandrodw3/liferay-portal/pull/117">sandrodw3#117</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/6418587">[master] ci:test:change-tracking-web-playwright - sandrodw3 &gt; sandrodw3 - PR#117 - 2024-09-10[03:07:57]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>6418587<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/32382/">publish-testray-report#32382</a><br/></div></div>

sandrodw3 committed:
LPD-25688 Use existing method that makes sure the web content editor is ready

sandrodw3 committed:
LPD-25688 This should be waitFor

isVisible just checks if the element is visible, it's thought to be used in if conditions

sandrodw3 committed:
LPD-25688 Use backendPage and move login fixture to tests themselves

sandrodw3 commented:
ci:test:change-tracking-web-playwright

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:change-tracking-web-playwright - 2 out of 2 jobs passed in 1 hour 19 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8abf22818849fef022c9ecc16232b3d4ea835385">8abf22818849fef022c9ecc16232b3d4ea835385</a></p><details><summary><strong>ci:test:change-tracking-web-playwright - 2 out of 2 jobs PASSED</strong></summary><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/10941/">test-portal-acceptance-pullrequest(master)</a></li><li>playwright-js-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/425762/">playwright-js-tomcat90-mysql57-jdk8/0/0</a></li></ul></li></ul></details></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10941/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/10941/">test-portal-acceptance-pullrequest(master)#10941</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10941/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>change-tracking-web-playwright<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/sandrodw3/liferay-portal/pull/117">sandrodw3#117</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/6846699">[master] ci:test:change-tracking-web-playwright - sandrodw3 &gt; sandrodw3 - PR#117 - 2024-09-10[07:15:48]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>6846699<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/23545/">publish-testray-report#23545</a><br/></div></div>
